### PR TITLE
Compare compatible objects (int lists) in assertDynamicFiltering

### PR DESCRIPTION
**Pull request by sopel39**

### CODE REVIEW

**Review by wendigo — APPROVED** [review]

File: presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduIntegrationDynamicFilter.java [hunk 1]
```diff
@@ -38,6 +38,7 @@
 import org.intellij.lang.annotations.Language;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.Test;
+import org.testng.internal.collections.Ints;
```
Comment (losipiuk) [inline presto-kudu/src/test/java/io/prestosql/plugin/kudu/TestKuduIntegrationDynamicFilter.java:41]: `org.testng.internal` ?

**Review by losipiuk — CHANGES_REQUESTED** [review]
I see that we use `org.testng.internal` in couple places. Would you have our own tool for converting int[] to List<Integer> and drop the use of internal package?

**Comment by sopel39** [comment]
@losipiuk it was meant to use `com.google.common.primitives.Ints;`

Thanks for catch!

**Comment by losipiuk** [comment]
> @losipiuk it was meant to use `com.google.common.primitives.Ints;`
> 
> Thanks for catch!

Cool. Can you fix other occurrences and add modernizer rule to build?

**Review by losipiuk — APPROVED** [review]

**Pull request merged**